apps/src/StudioApp.js

```diff
@@ -1793,7 +1794,7 @@ StudioApp.prototype.configureDom = function (config) {
   // TODO (cpirich): make conditional for applab
   var belowViz = document.getElementById('belowVisualization');
   var referenceArea = document.getElementById('reference_area');
-  if (referenceArea) {
+  if (!experiments.isEnabled('resourcesTab') && referenceArea) {
```

Comment:
For non starlab apps, do we still need to show videos in this reference area?

Reply:
The specification says only starlab apps.

Reply:
Right now, if I enable this experiment, isn't it going to also turn off videos that are in the resource tab for non-star lab apps?

Reply:
@epeach I think the above comment still needs to be addressed?